packages/rendermime/src/renderers.ts

```diff
@@ -135,6 +138,11 @@ namespace renderHTML {
      * Whether the node should be typeset.
      */
     shouldTypeset: boolean;
+
+    /**
+     * The Latex typesetter for the application.
```

Comment:
Perhaps, "The LaTeX typesetter" here as well?

Reply:
(same in other places in this file)

Reply:
I agree. Probably not for type and variable names, though?

Reply:
agreed